src/ray/core_worker/tests/BUILD.bazel

```diff
@@ -11,6 +11,18 @@ ray_cc_test(
     ],
 )
 
+ray_cc_test(
+    name = "shutdown_coordinator_test",
+    size = "medium",
+    srcs = ["shutdown_coordinator_test.cc"],
+    tags = ["team:core"],
+    deps = [
+        "//src/ray/core_worker:core_worker_lib",
+        "@com_google_googletest//:gtest",
+        "@com_google_googletest//:gtest_main",
+    ],
+)
+
```

Comment:
@codope this needs to be fixed to not pull in the entire `core_worker_lib`. Instead, it should depend on whatever specific sub-components are needed. In this case, that should be nearly none of them.

I'll merge the PR now to avoid slowing you down, but please follow up immediately to fix this and let me know if you have questions about how to do it.